### ISSUE TITLE
Can not detect object after training your network ?

### ISSUE THREAD

**Issue by ghost** [hizhangp/yolo_tensorflow#87]
Hello, I have modified your net to train only 2 objects which are dog and person, but after training process, I get the loss is around 6 and then I tested (inference) the pre-trained model, the net can not detect anything. Do you have encounter this problem ? If yes, please tell me where is wrong.
Anyway, thanks for your contribution.

**Comment by Benzzzxxx** [comment]
Maybe your train data is not enough or train epochs is not enough.

**Comment by LinzzMichael** [comment]
I have meet the same problem,but my loss was very high,it's about 10,how about yours?

**Comment by Unhapppy** [comment]
i met the same problem, have you solved it?

**Comment by Jeremy-lf** [comment]
so am i ,what's your solution?

**Comment by ghost** [comment]
i dont know, either

**Comment by eefresher** [comment]
i also have same problem ,my loss is acound 7

**Comment by mxazz123** [comment]
The loss is too high. Just uncomments the WEIGHTS in yolo/config file, and retrain it

**Comment by ghost** [comment]
thank you very much :D
